scalyr_agent/copying_manager/worker.py

```diff
@@ -392,6 +395,11 @@ def __init__(self, configuration, worker_config_entry, session_id, is_daemon=Fal
         self.__last_total_bytes_copied = 0
         self.__last_total_bytes_pending = 0
 
+        self._closed_files_checkpoints = {}  # type: Dict
+        self._closed_files_checkpoints_lock = threading.Lock()
+
+        self._closed_log_processors = []  # type: List
```

Comment:
Why is this needed?  Does it also require the lock to read/write it?

Reply:
I removed this variable, so now only the `_closed_files_checkpoints ` remains.